core/lib/protobuf_config/src/eth_sender.rs

```diff
@@ -109,6 +109,7 @@ impl ProtoRepr for proto::Sender {
                 .and_then(|x| Ok(proto::ProofLoadingMode::try_from(*x)?))
                 .context("proof_loading_mode")?
                 .parse(),
+            pubdata_sending_mode: PubdataSendingMode::Calldata,
```

Comment:
Should be loaded from protobuf config

Reply:
agreed, if we're okay with the enum then ill raise the pr in consensus repo